experimental/components/tests/displaynames/tests.rs

```diff
@@ -2,10 +2,12 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+extern crate alloc;
+
+use alloc::borrow::Cow;
```

Comment:
Tests always have `std`
```suggestion
use std::borrow::Cow;
```